code/obj/flock/structure/relay.dm

```diff
@@ -26,7 +26,8 @@ TYPEINFO(/obj/flock_structure/relay)
 	bound_y = -64
 	hitTwitch = FALSE
 	show_in_tutorial = TRUE
-	tutorial_desc = "Your goal and purpose. The Relay becomes closer to being real as you gain more compute, eventually becoming solid at 500 compute. You must then defend it while it charges before unleashing The Signal, and if you fail your consciousness will be destroyed."
+	// TODO: Reword to include all goals, why does this have to be const :(
```

Comment:
todo + incorrect compute value

Reply:
fixed, todo is there because i cant figure out how to make it dynamic should compute value be. it will be fine as 500 as is but it sucks to have to change several files to change the compute value entirely